src/transactions/cdp.js

```diff
@@ -892,7 +892,7 @@ export async function voteCDP(account_id, option1, option2) {
   params.fee = 0;
```

Comment:
You also need to change the "6" to a "7" in the note (i.e. the variable stringVal)